app/models/rent.rb

```diff
@@ -2,4 +2,15 @@ class Rent < ApplicationRecord
   belongs_to :user
   belongs_to :book
   validates :start_of_rent, :end_of_rent, presence: true
+  scope :on_date, -> { where('? BETWEEN start_of_rent AND end_of_rent', Date.today).where(returned_at: nil) }
+  scope :active_rentals, -> { on_date.order('end_of_rent ASC') }
+  before_create :upload_book
+
+  private
+
+  def upload_book
+    book_to_update = Book.find(self.book.id)
+    book_to_update.times_rented += 1
+    book_to_update.save
```

Comment:
```
book_to_update = self.book
book_to_update.times_rented += 1
book_to_update.save
```
The find is redundant

Reply:
Also, if you want to know the amount of times the book was rented, you have to only count the rents that exist for this book.

`self.book.rents.count`